docs/application-connector/docs/040-remote-evironment-custom-resource.md

```diff
@@ -54,16 +52,15 @@ This table lists all the possible parameters of a given resource together with t
 | Parameter   |      Mandatory?      |  Description |
 |:----------:|:-------------:|:------|
 | **metadata.name** |    **YES**   | Specifies the name of the CR. |
-| **spec.source** |    **NO**   | Identifies the Remote Environment in the cluster. |
-| **spec.source.environment** |    **YES**   | Specifies the environment of the connected Remote Environment. |
-| **spec.source.type** |    **YES**   | Specifies the type of the connected Remote Environment. |
-| **spec.source.namespace** |    **YES**   | Specifies the namespace of the connected Remote Environment. |
 | **spec.description** |    **NO**   | Describes the connected Remote Environment.  |
 | **spec.accessLabel** |    **NO**   | Labels the environment when the [EnvironmentMapping](041-environment-mapping-custom-resource.md) is created. |
 | **spec.services** |    **NO**   | Contains all services that the Remote Environment provides. |
 | **spec.services.id** |    **YES**   | Identifies the service that the Remote Environment provides. |
+| **spec.services.name** |    **YES**   | Specifies the CLI-friendly name of the Remote Environment service. MUST only contain lowercase characters, numbers and hyphens (no spaces). MUST be unique across all service objects returned in Remote Environment custom resource. MUST be a non-empty string. |
 | **spec.services.displayName** |    **YES**   | Specifies a human-readable name of the Remote Environment service. |
-| **spec.services.longDescription** |    **NO**   | Provides a human-readable description of the Remote Environment service. |
+| **spec.services.description** |    **YES**   | Specifies the short description of the Remote Environment service. MUST be a non-empty string. |
```

Comment:
`Provides a short description of the Remote Environment service. This string cannot be empty.`